setup.py

```diff
@@ -70,7 +69,7 @@ def get_version():
     install_requires=INSTALL_REQUIRES,
     tests_require=extras["test"],
     extras_require=extras,
-    keywords=["HEP", "particle", "decay", "representation"],
+    keywords=["HEP", "particle", "decay", "chain" "representation"],
```

Comment:
```suggestion
    keywords=["HEP", "particle", "decay", "chain", "representation"],
```

This is currently:
```python
    keywords=["HEP", "particle", "decay", "chainrepresentation"],
```

Which I don't think is what you meant.

Reply:
I think our flake8 (at least bugbear) testing would catch this.

Reply:
My eyes did not see the missing comma. Yes, that would be found, I guess too.